### DISCUSSION TITLE
How to delete old releases

### DISCUSSION THREAD

**Discussion by siddhrock** [deployphp/deployer#3413] [Help needed]
deploy:cleanup is not deleting old releases. Please suggest.

**Comment by antonmedv** [comment]
It should delete.

**Reply by siddhrock** [reply]
its not deleting. have you faced such issue? what could I do to fix it?

**Reply by antonmedv** [reply]
Try to debug.

**Reply by siddhrock** [reply]
I already tried to debug on it. can you suggest a way?

**Reply by antonmedv** [reply]
Deploy with -vvv arg.

**Reply by tormi** [reply]
[Default](https://github.com/deployphp/deployer/blob/master/recipe/common.php#L45-L46) is 10 releases, you can override the value as follows: `set('keep_releases', 3);`
